README.md

```diff
@@ -326,6 +327,19 @@ you can check the
 [vc-test-suite-implementations](https://github.com/w3c/vc-test-suite-implementations)
 library.
 
+### Allure Reporting
+It's also possible to generate local allure reports for analyzing and debuging results.
+
+To run the tests and browse the report, use the following commands:
+```bash
+# Running the tests
+npx mocha tests/
+
+# Running the reporting server
+allure serve allure-results
```

Comment:
Should this be run with `npx`?

Reply:
It's not necessary, the `allure-commandline` package provide a direct commandline access to the allure package:
https://www.npmjs.com/package/allure-commandline

There's 2 npm packages installed here, `allure-mocha` and `allure-commandline`. `allure-mocha` is the mocha specific reporter that will output the results into a directory. The `allure-commandline` is the allure software that will build the report and host a local reporting server.